make/dev.mk

```
@@ -45,11 +45,10 @@ dev.update-bootstrap-credentials-aws:
 
 .PHONY: release-please
 release-please:
-ifneq ($(GIT_CURRENT_BRANCH),main)
-	$(error "release-please should only be run on the main branch")
+ifneq ($(shell [[ "$(GIT_CURRENT_BRANCH)" = "main" ]] || [[ "$(GIT_CURRENT_BRANCH)" == release/v* ]]; echo $$?),0)
```

Instead of using `shell` here, let's use make's `filter` instead to not need to shell out:

```suggestion
ifeq ($(filter main release/v%,$(GIT_CURRENT_BRANCH)),)
```

---

fixed https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/912/commits/15279b83dc0cd138df520e5df8fdeffca6563da7